### ISSUE TITLE
How about dynamically generated GeoJSON-VT vector tiles?

### ISSUE THREAD

**Issue by t-sch** [SpatialServer/Leaflet.MapboxVectorTile#37]
I understand the mapbox vector tile layer reads pbf files. Why not generate them on-the-fly? https://www.mapbox.com/blog/introducing-geojson-vt/

**Comment by bwyss** [comment]
+1

**Comment by henryptung** [comment]
More generally, I would vote for a provider interface over bundling GeoJSON-VT. Honestly, I think the library could benefit from an API between data fetching/parsing and rendering, something like `getFeatures(z, x, y): VectorTileFeature[]`. This would make it trivial to write different data providers, including a thin wrapper for a GeoJSON-VT source.

**Comment by wilg** [comment]
I agree, my use case involves both on-the-fly and precalculated vector tiles.

**Comment by reyemtm** [comment]
+1 for geojson-vt compatibility
